### DISCUSSION TITLE
Hammer Dart Traps Permissions

### DISCUSSION THREAD

**Discussion by chris-der-kleine** [Pryaxis/TShock#2077] [Q&A]
Hey,
I host a TShock Server with the current Tshock 4.4.0 Pre 12 Version on 1.4.0.5.

I noticed that as Superadmin I'm able to rotate dart traps without any problems.  
But non-superadmins can't rotate dart traps. And there seems to be no error message about anything.

(SSC is enabled, if this may help)

Does anyone know which permission is responsible for this?

**Comment by hakusaro** [comment]
Playing as superadmin is really unsupported. Play as owner and see if that makes things more normal. I'm guessing it's the send tile square system breaking here, but I'm not fully sure. @QuiCM do you have any ideas off the top of your head?
